helpers/formatHelpers.js

```diff
@@ -142,16 +160,7 @@ function formatSgDataProducts(products) {
   if (!products || !Array.isArray(products)) {
     return [];
   }
-
-  return products.map(product => ({
-    id: getProductIdentifier(product),
-    type: 'product',
-    name: get(product, 'name'),
-    priceNet: getUnifiedNumber(get(product, 'amount.net')),
-    priceGross: getUnifiedNumber(get(product, 'amount.gross')),
-    quantity: getUnifiedNumber(get(product, 'quantity')),
-    currency: get(product, 'amount.currency'),
-  }));
+  return products.map(product => formatSgDataProduct(product));
```

Comment:
Could be just `return products.map(formatSgDataProduct);` I think

Reply:
done